web/src/components/itemList.js

```diff
@@ -113,15 +97,17 @@ class ItemList extends Component {
                        onCompleted={data => {
                            const selected = [];
                            const currency = [];
+                           
                            for(let i = 0; i < data.getItems.length; i++) {
-                               selected.push(0);
+                               selected.push(data.getItems[i].select);
```

Comment:
All the other information about items are not updated by this process. This is the reason why lines 26, 28 are not working.